### DISCUSSION TITLE
Disabling SEO features

### DISCUSSION THREAD

**Discussion by diogovitzo** [lucky-media/landtamic#24] [General]
Hi, 

I'm currently working on a Statamic website with the Landtamic starter kit, and we've now reached the stage in which we're starting to look into SEO optimizations and features.

With that in mind, we were considering purchasing one of the available SEO add-ons (such as aardvark's), but I'm not sure to what extent there might be a conflict with the SEO features that Landtamic provides. Is there any way we can disable all the Landtamic SEO features, and/or do they integrate nicely?

Thanks!

**Comment by lokmanm** [comment]
Hey @diogovitzo,

We provide a minimal config that's very easy to replace if you decide to go with another add-on.

Here are the steps needed to remove the default SEO:

1. Remove the SEO Fieldset from your collections: pages, blog, etc.
2. Remove the [SEO Partial](https://github.com/lucky-media/landtamic/blob/8a4b954ec9c49656e7d8e53c7a57c408cd06ea0d/resources/views/layout.antlers.html#L13)

That's it. Now you can install any SEO add-on and follow the instructions there.

**Reply by diogovitzo** [reply]
@lokmanm Thanks! Can I also delete the SEO Global? I assume so just making sure.
